ADDRESSING.md

```diff
@@ -20,16 +18,21 @@
 
 ## TL;DR
 
-Native URI can be used for addressing:
-- immutable content:  `ipfs://{cid}`
-- mutable content behind IPNS record signed with a specific libp2p key: `ipns://{libp2p-key}` 
-- mutable content behind human-readable DNSLink (DNS TXT record): `ipns://{fqdn}` 
+For immutable content use an IPFS URI (`ipfs://{cid}`).
+- [ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy](ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy)
+
+For mutable content use an IPNS DNSLink URI (`ipns://{fqdn}`).
+- [ipns://webui.ipfs.io](ipns://webui.ipfs.io)
```

Comment:
```suggestion
- [ipns://cid.ipfs.io](ipns://cid.ipfs.io)
- [ipns://k51qzi5uqu5dgutdk6i1ynyzgkqngpha5xpgia3a5qqp4jsh0u4csozksxel2r](ipns://k51qzi5uqu5dgutdk6i1ynyzgkqngpha5xpgia3a5qqp4jsh0u4csozksxel2r)
```

I think it is better if we add it, even if this is this silly example for now.
The risk of cryptographic identifiers being ignored in implementations is too high.